src/Controller/GeneratePdf.php

```diff
@@ -58,13 +62,14 @@ public function handle(): Response
 
         $pdf = $this->pdfCreator->generatePdf($template);
         $dateAtom = (new \DateTime())->format(\DateTime::ATOM);
+        $eventName = $this->eventService->getSelectedEvent()->getName();
```

Comment:
Because you're only using a single event (an event name even), you could only inject that specific bit.
In order to do that, you can do a `CurrentEventInterface` somewhere, and create an entry in `services.yaml` to wire it (using the factory syntax).